src/Compilers/Core/CodeAnalysisTest/MetadataReferences/ModuleMetadataTests.cs

```diff
@@ -77,18 +75,20 @@ public void CreateFromImageStream()
             Assert.Throws<ArgumentException>(() => ModuleMetadata.CreateFromStream(new TestStream(canRead: true, canSeek: false)));
         }
 
-        [Fact]
+        [ConditionalFact(typeof(DesktopOnly))]
```

Comment:
This will eventually need to be Windows Desktop only correct? As defined it runs on any desktop runtime including Mono.

Reply:
Yes, that's probably true. I didn't try running these tests with Mono but I expect they'll have problems.